server/testdata/tools/mistral.gotmpl

```diff
@@ -1,11 +1,12 @@
 {{- range $index, $_ := .Messages }}
 {{- if eq .Role "user" }}
-{{- if and (eq (len (slice $.Messages $index)) 1) $.Tools }}[AVAILABLE_TOOLS] {{ $.Tools }}[/AVAILABLE_TOOLS]
-{{- end }}[INST] {{ if and (eq (len (slice $.Messages $index)) 1) $.System }}{{ $.System }}
+{{- $last := (or (eq (len (slice $.Messages $index)) 1) (eq (len (slice $.Messages $index)) 2)) }}
```

Comment:
This depends on messages being collated, maybe there's a better way than checking if it's the previous two (i.e. last message is user OR last two messages are user-assistant)